src/main/resources/wa-task-completion-sscs-benefit.dmn

```diff
@@ -119,6 +119,17 @@
           <text>"Auto"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_0d06llx">
+        <inputEntry id="UnaryTests_16nn4kn">
+          <text>"reviewPheRequest","interlocReviewStateAmend"</text>
```

Comment:
Needs to be "reviewPhmeRequest".  The event id is slightly different to the task id, confusingly!

Reply:
Well spotted, I have updated. Thanks.